packages/shared/src/components/auth/LoginForm.tsx

```diff
@@ -108,6 +111,21 @@ function LoginForm({
           {loginButton}
         </Button>
       </span>
+      {hint && hint === labels.auth.error.invalidEmailOrPassword && (
```

Comment:
This is a bit of a mission, but the request was to make it more clear so a custom hint exception for this specific hint only

Reply:
assuming `labels.auth.error.invalidEmailOrPass` is never null or undefined, I think there would be no need to check if `hint` exists as well
```suggestion
      {hint === labels.auth.error.invalidEmailOrPassword && (
```